content/posts/jenkins-x-flow-integration.md

```diff
@@ -0,0 +1,147 @@
+---
+author:
+  name: "Josh Hendrick"
+title: "Extending Jenkins X for Traditional Deployments with CloudBees Flow"
+date: 2019-05-28T10:58:46-04:00
+showDate: true
+draft: true
+tags: ["jenkins","jenkins x","cloudbees flow", "tekon", "extending jenkins x"]
+---
+[Jenkins X](https://jenkins-x.io) is quickly becoming the de facto standard for high performing teams wanting to do CI/CD in a highly scalable and fault tolerant environment. For those who haven’t gotten the opportunity to try out Jenkins X, it allows teams to run CI/CD workloads natively in a Kubernetes environment while taking advantage of modern operating patterns like GitOps and serverless architectures. For teams wanting to modernize their continuous integration and continuous deployment capabilities, Jenkins X is the go to solution.
+
+In today’s heterogenous technology environment, most organizations tend to have a mix of modern cloud native architectures as well as more traditional workloads which get deployed either on-prem or within the cloud. In the latter case, a combination of Jenkins X (performing CI steps) and CloudBees Flow (handling the deployment) can add a huge amount of flexibility and power to a Continuous Delivery process.  Jenkins X can be easily extended to accommodate any type of workload required - it can be a full end to end CI/CD tool for building, deploying, and running applications all within a Kubernetes cluster, or it can handle CI while offloading other release and deployment tasks to another solution.  In this blog post we’re going to cover how Jenkins X can be extended to offload release/deployment tasks to [CloudBees Flow](https://www.cloudbees.com/cloudbees-acquires-electric-cloud).
```

Comment:
Haven't read the whole thing yet, but would be cool to talk about visibility across the application landscape as well.